opencv/src/OpenCV/Core/Types.hsc

```diff
@@ -212,6 +216,14 @@ instance FromPtr KeyPoint where
 instance CSizeOf C'KeyPoint where
     cSizeOf _proxy = c'sizeof_KeyPoint
 
+
+instance Storable KeyPoint where
+    sizeOf _ = c'sizeof_KeyPoint
+    alignment _ = 8 -- ???
```

Comment:
Uhm.. This is tricky :-)

I just noticed that c++11 has an [alignof](http://en.cppreference.com/w/cpp/language/alignof) operator. We could use that to determine the proper alignment.

I'm thinking about another [HSC macro](https://github.com/LumiGuide/haskell-opencv/blob/master/opencv/include/hsc_macros.hpp) like `hsc_sizeof`.

Writing this `hsc_alignof` macro is mostly copy+pasting the `hsc_sizeof` macro.